### ISSUE TITLE
How to delete a comment in live preview?

### ISSUE THREAD

**Issue by lalulalulearing** [Fevol/obsidian-criticmarkup#20]
Right now it seems a comment can only be deleted by switching to the source mode? Is there any way to delete it directly in the live preview? Or could this function be added please?

**Comment by zhampu** [comment]
Im also interested in know this...

**Comment by kerim** [comment]
This is related to: https://github.com/Fevol/obsidian-criticmarkup/issues/17

**Comment by Fevol** [comment]
Yes, this is indeed the same problem as #17, I'll close this issue in favor of that one for further discussion.
